ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2017,6 +2017,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: baremetal-upi-ovn-ipv4-disconnected-proxy-p1-f14
```

Comment:
can we move this to the multi file and set it for 4.16? We could have a multi-payload-based job on amd64 only for this configuration.